app/assets/stylesheets/cards.css.scss

```diff
@@ -0,0 +1,8 @@
+table.cards {
+  border: 1px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.